backend/src/main/java/com/bakdata/conquery/apiv1/query/concept/specific/external/DateFormat.java

```diff
@@ -0,0 +1,55 @@
+package com.bakdata.conquery.apiv1.query.concept.specific.external;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import com.bakdata.conquery.models.common.CDateSet;
+import com.bakdata.conquery.models.common.daterange.CDateRange;
+import com.bakdata.conquery.models.preproc.parser.specific.DateRangeParser;
+import com.bakdata.conquery.util.DateReader;
+
+public enum DateFormat {
+	EVENT_DATE {
+		@Override
+		public void readDates(String value, DateReader dateReader, CDateSet out) {
+			out.add(CDateRange.exactly(dateReader.parseToLocalDate(value)));
+		}
+	},
+	END_DATE {
+		@Override
+		public void readDates(String value, DateReader dateReader, CDateSet out) {
+			out.add(CDateRange.atMost(dateReader.parseToLocalDate(value)));
+		}
+	},
+	START_DATE {
+		@Override
+		public void readDates(String value, DateReader dateReader, CDateSet out) {
+			out.add(CDateRange.atLeast(dateReader.parseToLocalDate(value)));
+		}
+	},
+	DATE_RANGE {
+		@Override
+		public void readDates(String value, DateReader dateReader, CDateSet out) {
+			out.add(DateRangeParser.parseISORange(value, dateReader));
+		}
+	},
+	DATE_SET {
+		@Override
+		public void readDates(String value, DateReader dateReader, CDateSet out) {
+			out.addAll(CDateSet.parse(value, dateReader));
+		}
+	},
+	ALL {
+		@Override
+		public void readDates(String value, DateReader dateReader, CDateSet out) {
+			out.add(CDateRange.all());
+		}
+	};
```

Comment:
Ist das ein valider Modus? Er ignoriert ja gegebene Werte. Das sollte im dokumentiert sein.

Reply:
der ist dafür da falls nichts ausgewählt wird

Reply:
Aber wird der aktiv genutzt? In CQExternal#readDates sieht es so aus als würde der ALL Fall separat erzeugt werden (wenn kein Datumsformat gefunden wurde)

Reply:
Verstehe nicht was du meinst, ich benutze ihn um so eine globale spanne zu bekommen. Theoretisch könnte man eine Query damit basteln, die wäre halt genau so sinnvoll wie eine Query die keine Datumsspalten enthält.

Reply:
Ah okay, ich sehe jetzt wo das Problem ist. Ich habe aber deinen Kommentar  
> der ist dafür da falls nichts ausgewählt wird
nicht so richtig verstanden. Wo wird nichts ausgewählt?
Wird ALL, eigentlich dem Frontend angeboten?

Reply:
Leider hatte Kai nicht genug Zeit das Protokoll so umzusetzen, deswegen sind die Datumsfelder doch hard-coded auf Frontend Seite.